### PR TITLE
[swift-4.0-branch][overlay] Adding missing files to Intents/CMakeLists.txt

### PR DESCRIPTION
**Pull request by moiseev**
<rdar://problem/32901674>

### CODE REVIEW

**Comment by moiseev** [comment]
@swift-ci Please test macOS platform

**Comment by moiseev** [comment]
@swift-ci Please test Linux platform

File: stdlib/public/SDK/Intents/INSearchCallHistoryIntent.swift [hunk 1]
```diff
@@ -14,29 +14,25 @@
 import Foundation
 
 #if os(iOS) || os(watchOS)
-@available(iOS 10.0, watchOS 3.2, *)
+@available(iOS 11.0, watchOS 4.0, *)
 extension INSearchCallHistoryIntent {
+  @available(iOS 11.0, watchOS 4.0, *)
```
Comment (jrose-apple) [inline stdlib/public/SDK/Intents/INSearchCallHistoryIntent.swift]: This is a behavior change, even if it wasn't working before. Do we really not want to support this entry point on pre-iOS-11 systems?

Comment (jrose-apple) [inline stdlib/public/SDK/Intents/INSearchCallHistoryIntent.swift]: (Or rather, do the Intents folks not want to support this etc etc.)

Comment (moiseev) [inline stdlib/public/SDK/Intents/INSearchCallHistoryIntent.swift]: FWIW, the patch originated from the Intents team. 🤷‍♂️

**Comment by moiseev** [comment]
@swift-ci Please test

**Pull request merged**